.gitignore

```diff
@@ -395,6 +395,7 @@ FodyWeavers.xsd
 *.msp
 
 # JetBrains Rider
+.idea
```

Comment:
This is one to discuss further - I put it in temporarily to prevent my personal settings from getting committed but it needs a wider conversation about what value the IDE settings bring, since in the future, its quite feasible the support team won't be using Rider.

Reply:
Great - I've simplified the gitIgnore and left the .idea entry.